.eslintrc.yml

```diff
@@ -95,22 +96,33 @@ rules:
     - error
     - allowExpressions: true
   react/react-in-jsx-scope: 0 # not needed because React 17 fixed this!
+  unicorn/no-null: 0
+  unicorn/prefer-node-protocol: 0
+  unicorn/filename-case: 0
+  unicorn/no-array-for-each: 0
+  unicorn/no-array-reduce: 0
+  unicorn/no-array-callback-reference: 0
 overrides:
   - files:
       - '**/*.js'
     rules:
       '@typescript-eslint/explicit-function-return-type': 0
       '@typescript-eslint/explicit-module-boundary-types': 0
-      'react/jsx-filename-extension': 0
+      react/jsx-filename-extension: 0
       '@typescript-eslint/no-unsafe-return': 0
       '@typescript-eslint/no-unsafe-assignment': 0
       '@typescript-eslint/no-unsafe-call': 0
       '@typescript-eslint/no-unsafe-member-access': 0
       '@typescript-eslint/no-var-requires': 0
+      unicorn/prefer-module: 0
   - files:
       - 'scripts/**/*.{ts,tsx,js,jsx}'
     rules:
       no-console: 0
+  - files:
+      - 'next-env.d.ts'
+    rules:
+      unicorn/prevent-abbreviations: 0
```

Comment:
if we want to disable a single rule for a single file, how about we do it using `eslint-disable`? I feel adding single rule exceptions for individual files might increase the size of eslintrc file and we might want to keep it only for maintaining all generic rules. Not sure though what's the best practice here?

Reply:
yeah, makes total sense. Just pushed this change.